conf/application.conf

```diff
@@ -20,7 +20,7 @@ play.http.router = prod.Routes
 play.filters.enabled += play.filters.csp.CSPFilter
 
 # Default http client
-play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientV2Module"
```

Comment:
Hi @kenneth-rayner, can you please rollback this change in your PR? PlatUI will be looking to upgrade the `HttpClient` module used, but we will want to do this within our team and ensure it all works as expected.